azure/template.json

```diff
@@ -30,6 +30,9 @@
         "databaseTier": {
             "type": "string"
         },
+        "databaseSizeBytes": {
+            "type": "string"
+        },
```

Comment:
Please can you check why Azure/template.json is appearing in the Pull as File Changed. This shouldn't appear as it's already in master.